Sources/Mistica/Components/Crouton/Domain/CroutonConfig.swift

```diff
@@ -17,6 +17,8 @@ import UIKit
 public enum CroutonDismissInterval: Int, CaseIterable {
     case fiveSeconds = 5
     case tenSeconds = 10
+    // The infinite rawValue will/should not be used to dispatch a timer.
+    case infinite = 999
 }
```

Comment:
I would do:
```suggestion
public enum CroutonDismissInterval: CaseIterable {
    case fiveSeconds
    case tenSeconds
    case infinite
}

public extension CroutonDismissInterval {
    var timeInterval: TimeInterval? {
        switch self {
            case .fiveSeconds:
                return 5
            case .tenSeconds:
                return 10
            case .infinite:
                return nil
        }
    }
}
```

and then
```diff
    func addCountdownToDismiss() {
-        guard config.overrideDismissInterval != .infinite else {
+        guard let timeInterval = config.overrideDismissInterval.timeInterval else {
            return
        }

        timer = Timer.scheduledTimer(
-            timeInterval: config.overrideDismissInterval.timeInterval,
+            timeInterval: timeInterval,
            target: self,
            selector: #selector(invokeDismissHandler),
            userInfo: nil,
            repeats: false
        )
    }
```

Reply:
Fixed: https://github.com/Telefonica/mistica-ios/pull/322/commits/7d25de85d99078554f37a048c699a4a9f9b0710b